arangod/Pregel/ArangoExternalDispatcher.h

```diff
@@ -0,0 +1,96 @@
+#pragma once
+
+#include "Actor/ActorPID.h"
+#include "Actor/Message.h"
+#include "CrashHandler/CrashHandler.h"
+#include "Assertions/ProdAssert.h"
+#include "Inspection/VPackWithErrorT.h"
+#include "Logger/LogMacros.h"
+#include "Network/ConnectionPool.h"
+#include "Network/Methods.h"
+
```

Comment:
There should be some documentation here about what the responsibility of this ArangoExternalDispatcher is